packages/notebook/src/windowing.ts

```diff
@@ -61,9 +61,12 @@ export class NotebookViewModel extends WindowedListModel {
     if (model instanceof CodeCellModel) {
       for (let outputIdx = 0; outputIdx < model.outputs.length; outputIdx++) {
         const output = model.outputs.get(outputIdx);
-        outputsLines = ((output.data['text/plain'] as string) ?? '').split(
-          '\n'
-        ).length;
+        const data = output.data['text/plain'];
+        if (typeof data === 'string') {
+          outputsLines += data.split('\n').length;
+        } else if (Array.isArray(data)) {
+          outputsLines += data.join('\n').split('\n').length;
```

Comment:
I need to double check this but I think the new lines should be included in the array.

```suggestion
          outputsLines += data.join('').split('\n').length;
```